rust/index/Cargo.toml

```diff
@@ -19,6 +19,7 @@ tracing = { workspace = true }
 roaring = { workspace = true }
 futures = { workspace = true }
 async-trait = { workspace = true }
+tempfile = { workspace = true }
```

Comment:
I think this should probably stay as a dev dependency?

Reply:
See below